Phase04/CalculatorTester.cs

```diff
@@ -0,0 +1,74 @@
+using SimpleCalculator.Business;
+using SimpleCalculator.Business.Enums;
+using SimpleCalculator.Business.Abstraction;
+using SimpleCalculator.Business.OperatorBusiness;
+using SimpleCalculator.Business.OperatorBusiness.Operators;
+
+public class CalculatorTester
+{
+    private const int MaxInt = 2147483647;
+    private const int MinInt = -2147483648;
+    private readonly Calculator _calculator;
+    private readonly OperatorProvider _provider;
+    private readonly SumOperator _sumOperator;
+    private readonly SubOperator _subOperator;
+    private readonly MultiplyOperator _multiplyOperator;
+    private readonly DivisionOperator _divisionOperator;
+    public CalculatorTester() {
+        this._calculator = new Calculator();
+        this._provider = new OperatorProvider();
+        this._sumOperator = new SumOperator();
+        this._subOperator = new SubOperator();
+        this._divisionOperator = new DivisionOperator();
+        this._multiplyOperator = new MultiplyOperator();
+    }
+
+    [Theory]
+    [InlineData(0,1,1)]
+    [InlineData(2,0,2)]
+    [InlineData(-3,0,-3)]
+    public void CalculatorAdditionTest(int firstOperand, int secondOperand, int expected) {
+        var provider_sub = new Mock<IOperatorProvider>();
+        provider_sub.Setup(x => x.GetOperator(OperatorEnum.sum)).Returns(new SumOperator());
+        var calculator = new Calculator(provider_sub.Object);
```

Comment:
_calculator is already initialized and can be used here.
the is no need for new calculator.